packages/core/test/actions.test.ts

```diff
@@ -39,7 +39,6 @@ describe('entry/exit actions', () => {
   };
 
   const lightMachine = createMachine({
-    key: 'light',
```

Comment:
We should preserve these as `id: 'light',` etc.

Reply:
I've already done this in places where it mattered for the tests. Do you want me to adjust other places too?